pkg/database/database.go

```diff
@@ -27,7 +28,7 @@ var RegisteredDBCreators = make(map[string]newDBCreatorFunc)
 type DBInterface interface {
 	QueryResources(ctx context.Context, kind, apiVersion, namespace,
 		name, continueId, continueDate string, limit int) ([]string, int64, string, error)
-	QueryLogURLs(ctx context.Context, kind, apiVersion, namespace, name string) ([]string, error)
+	QueryLogURL(ctx context.Context, kind, apiVersion, namespace, name string) (string, error)
```

Comment:
The same question again about getting the log for a specific container.

Reply:
Same answer: 

Yes, we need to allow the user to specify which container they want but I think we also should work when no container is specified and default to return one of them (replicating the kubectl logs command behavior).

For introducing the container filtering there is a separate issue - https://github.com/kubearchive/kubearchive/issues/631